src/m365/spo/commands/list/list-view-remove.ts

```diff
@@ -145,7 +156,7 @@ class SpoListViewRemoveCommand extends SpoCommand {
         type: 'confirm',
         name: 'continue',
         default: false,
-        message: `Are you sure you want to remove the view ${args.options.viewId || args.options.viewTitle} from the list ${args.options.listId || args.options.listTitle} in site ${args.options.webUrl}?`
+        message: `Are you sure you want to remove the view from the list?`
```

Comment:
Is there a reason you removed the view/list information from the message? Let's keep it in there to have a better experience for the end user.